sdk/storage/Azure.Storage.Blobs/src/Azure.Storage.Blobs.csproj

```diff
@@ -19,7 +19,7 @@
       REST API Reference for Blob Service - https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api
     </Description>
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
-    <IsTrimmable Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))">true</IsTrimmable>
+    <IsTrimmable Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">true</IsTrimmable>
```

Comment:
As I mentioned in the issue, do not change IsTargetFrameworkCompatible statements, revert this change.